src/main/java/nextstep/subway/path/ui/PathController.java

```diff
@@ -17,7 +19,7 @@ public PathController(PathService pathService) {
     }
 
     @GetMapping("/paths")
-    public ResponseEntity<PathResponse> findPath(@RequestParam Long source, @RequestParam Long target, @RequestParam PathType type) {
-        return ResponseEntity.ok(pathService.findPath(source, target, type));
+    public ResponseEntity<PathResponse> findPath(@AuthenticationPrincipal LoginMember loginMember, @RequestParam Long source, @RequestParam Long target, @RequestParam PathType type) {
```

Comment:
현재는 로그인이 필요한 요청에 비로그인 상태로 요청을 하더라도 401이 응답이 안되지 않나요?
로그인이 필수인 경우와 선택인 경우를 이런식으로 구분지으면 어떨까요?

```suggestion
    public ResponseEntity<PathResponse> findPath(@AuthenticationPrincipal(required = false) LoginMember loginMember, @RequestParam Long source, @RequestParam Long target, @RequestParam PathType type) {
```